src/theory/arith/nl/cad/cdcac.h

```diff
@@ -57,6 +57,12 @@ class CDCAC
   /** Collect variables from the constraints and compute a variable ordering. */
   void computeVariableOrdering();
 
+  /**
+   * Extract an initial assignment from the given model.
```

Comment:
Can you make the doc mention ran_variable?  This method is computing the model for that variable?  And storing it internally?